clusterloader2/testing/load/config.yaml

```diff
@@ -48,12 +48,34 @@
 {{$mediumDeploymentsPerNamespace := SubtractInt $mediumDeploymentsPerNamespace (IfThenElse $ENABLE_JOBS 1 0)}}
 {{$bigDeploymentsPerNamespace := SubtractInt $bigDeploymentsPerNamespace (IfThenElse $ENABLE_JOBS 1 0)}}
 
+# BEGIN scheduler-throughput section
+# Min number of pods per deployment to be used for measuring scheduler throughput
+# to get enough samples and accurate measurements in small clusters.
+{{$MIN_PODS_PER_DEPLOYMENT_TO_MEASURE_SCHEDULER_THROUGHPUT := 250}}
+# Number of namespaces with scheduler throughput deployments. This is also number
+# of scheduler throughput deployments as we create 1 such deployment per namespace.
+{{$schedulerThroughputNamespaces := 2}}
+{{$schedulerThroughputPodsPerDeployment := MaxInt $MIN_PODS_PER_DEPLOYMENT_TO_MEASURE_SCHEDULER_THROUGHPUT .Nodes}}
+{{$schedulerThroughputThreshold := DefaultParam .CL2_SCHEDULER_THROUGHPUT_THRESHOLD 0}}
+# END scheduler-throughput section
+
+# BEGIN pod-startup-latency section
+# Min number of pods to be used for measuring pod startup latency to get enough
+# samples and accurate measurements in small clusters.
+{{$MIN_PODS_TO_MEASURE_STARTUP_LATENCY := 500}}
```

Comment:
Can we make this a more generic constant?
And use it both to determine min number of latency pods as well as min number of pods needed for scheduler throughput?